### PR TITLE
backend-structure complete, under PR review

### PR DESCRIPTION
**Pull request by ZackRohde**

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1,7 +1,31 @@
...
+    image: 'https://images.unsplash.com/photo-1492370284958-c20b15c692d2?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1049&q=80'
+  }
+]
+
+cats.each do |each_cat|
+  Cat.create each_cat
+  puts "creating cat #{each_cat}"
+end
+
+Cats.create(name:"Fluffy", age:3, enjoys:"chasing rats", image:"dfhi")
```
Comment (triciasykes) [inline db/seeds.rb]: Line 31... you don't need to do this.  Just checking if there's a reason you added it?

**Review by triciasykes — CHANGES_REQUESTED** [review]
See the comment below about seeds file

File: db/seeds.rb (hunk 1 above)
Comment (ZackRohde) [reply db/seeds.rb]: We were testing it to make sure we could create, like the story said.

Comment (triciasykes) [reply db/seeds.rb]: You want to run that line in the rails console.  It shouldn't be in your program.  :)

Comment (dthomas91) [reply db/seeds.rb]: Change has been made and pushed to the branch

**Review by triciasykes — APPROVED** [review]

**Pull request merged**